src/VisualStudio/Core/Test/ChangeSignature/ChangeSignatureViewModelTests.vb

```diff
@@ -114,6 +114,44 @@ class MyClass
             monitor.Detach()
         End Function
 
+        <Fact, Trait(Traits.Feature, Traits.Features.ChangeSignature)>
+        Public Async Function ReorderParameters_MethodWithTwoDiscardParameters_MoveFirstParameterDown() As Tasks.Task
+            Dim markup = <Text><![CDATA[
+class MyClass
+{
+    public void $$M(int _, string _)
+    {
+    }
+}"]]></Text>
+
+            Dim viewModelTestState = Await GetViewModelTestStateAsync(markup, LanguageNames.CSharp)
```

Comment:
@dpoeschl Is this the right type of test for this?

Reply:
This test looks good to me.

Reply:
Although there should probably be another test added to the regular ChangeSignatureTests.cs that actually runs the signature change (this one here just makes sure the dialog won't choke on it).

Reply:
I tried to play with this and couldn't get it to work. Talked to @333fred and confirmed that this feature is only supposed to be for lambdas. Should this test be changed to an actual lambda change signature?

Reply:
The feature briefly supported discards in method signatures. I'll remove this test since the scope is now limited to lambdas and anonymous methods.
Added the test you suggested. Thanks